components/ThemeSwitcher/ThemeSwitcher.tsx

```diff
@@ -90,7 +90,9 @@ const ThemeSwitcher = () => {
 							Light mode
 						</button>
 					</li>
-					<li className={theme === "dark" ? styles.activeBtn : ""}>
+					<li
+						className={theme === "dark" ? styles.activeBtn : ""}
+						onBlur={handleThemeSwitcher}>
```

Comment:
This works great when tabbing forwards. However, if I tab to the dark theme then decide to shift tab back up it closes the pop up as the dark theme blurs which is not the expected behaviour so this might need a bit more work.

Things to test:
It closes when tabbing forwards past dark theme
It doesn't close when tabbing to dark theme then shift + tabbing back up through the list
It closes when tabbing to dark theme then using the mouse to make another selection
It closes when tabbing to another theme then using the mouse to select dark theme

Reply:
You're right! I tested only one way. I hope this is ok now.